src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -309,6 +363,47 @@ public Jwt parse(String jwt) throws ExpiredJwtException, MalformedJwtException,
         }
     }
 
+    private void validateExpectedClaims(Header header, Claims claims) {
+        for (String expectedClaimName : expectedClaims.keySet()) {
+            Object expectedClaimValue;
+            Object actualClaimValue;
+
+            // since issued at is a date, call the specific method
+            // other methods deal with strings and the more
+            // general method can be used
```

Comment:
Can't do this - users can expect a custom data field, and we have to handle that correctly as well.  i.e. we shouldn't have special logic for just one date field - any number of date fields could be present.